portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,8 +57,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long timestamp = (long) entity.getProperty("timestamp");
       String ipAddress = (String) entity.getProperty("ipAddress");
 
-      Comment newComments = new Comment(id, content, timestamp, ipAddress);
-      comments.add(newComments);
+      //if it is over max size, stop adding comments in
+      if (comments.size() >= maxNum) {
+          break;
+      } else {
```

Comment:
Since the if condition being true will cause the remaining instructions in the loop to be skipped, we can leave the code out of an else{} block.

It helps to reduce the nesting level of the code, which makes it more readable, and supports better maintainability.